site/content/en/docs/Guides/Client SDKs/sdk-server.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Agones SDK Server"
+linkTitle: "SDK Server"
+date: 2024-04-17
+weight: 1001
+description: "The SDK Server is a sidecar for a GameServer that will update the GameServer Status on SDK requests."
+---
+
+## SDK Server Overview
+- The SDK Server is a gRPC server. The methods for communication between the SDK Client and SDK Server are defined in the [SDK proto](https://github.com/googleforgames/agones/blob/main/proto/sdk).
+- The REST API is generated directly from the SDK .proto files.
+- All other SDK Client APIs are wrappers on top of the SDK proto definitions.
+
+{{% feature publishVersion="1.43.0" %}}
+
+{{< alert title="Note" color="info">}}
+This section is specifically about the SDK proto compatibility between Agones versions and deprecation policies.
+{{< /alert >}}
+
+## SDK Proto Compatibility Guarantees
+In order to allow compatibility between game server binaries and sdk-server, a game server binary using Beta and Stable SDK protos must remain compatible with a newer sdk-server.
+- Our SDK Server compatibility contract: If your game server uses a non-deprecated Stable API, your binary will be compatible for 10 releases (~1y) starting from the SDK version packaged
+  - For example, if the game server uses non-deprecated stable APIs in the 1.40 SDK, it will be compatible through the 1.50 SDK.
+  - Stable APIs will almost certainly be compatible beyond 10 releases, but 10 releases is guaranteed.
+- Similar to the compatibility contract for the Stable API, using a non-deprecated Beta API, your binary will be compatible for 5 releases (~6mo).
+- Alpha SDK Protos (and APIs/RPCs) are subject to change between releases.
+  - A game server binary using Alpha SDKs may not be compatible with a newer sdk-server.
+  - In Alpha, incompatible changes retaining the same SDK proto message name are allowed.
+  - When we make incompatible Alpha changes, we will document the APIs involved.
+
+## SDK Deprecation Policies
+Breaking changes will be called out in upgrade documentation, alongside the build horizon, allowing operators to plan their upgrades.
+### Stable Deprecation Policies
+A Stable API may be marked as deprecated in release X and removed from Stable in release X+10.
+TODO: Does this deprecation first need to flow through Alpha and / or Beta?
+### Beta Deprecation Policies
+When a feature graduates from Beta to Stable at release X the API is in both protos from release X to release X+5. The Beta API is marked as deprecated in release X and removed from Beta in release X+5.
```

Comment:
Not sure I entirely understood this comment?

Reply:
Instead of..
`When a feature graduates from Beta to Stable at release X the API is in both protos from [...]`

I was thinking more something like:
`When a feature graduates from Beta to Stable at release X, the API will be present in both Beta and Stable surfaces from [...]`